deployments/hadoop-yarn/ansible/33-install-monitor.yml

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
I had it as install-prometheus at first locally, but ended up with monitor because this installs more than just Prometheus (Prometheus / Grafana ..) In any case I'm fine with "install-prometheus.yml" so I'll update to that.